package.json

```diff
@@ -49,12 +51,15 @@
     "@babel/eslint-parser": "7.13.4",
     "@babel/preset-env": "7.13.5",
     "@babel/runtime": "7.13.6",
+    "@types/node": "20.8.10",
     "babel-eslint": "10.1.0",
     "eslint": "7.20.0",
     "eslint-plugin-instawork": "file:.",
     "jest": "26.6.3",
     "prettier": "2.2.1",
     "pretty-quick": "3.1.0",
-    "react": "17.0.1"
+    "react": "17.0.1",
+    "ts-node": "^10.9.1",
+    "typescript": "<5.0.0"
```

Comment:
We use typescript 5.2.2 on mobile and Hyperview repos, this will add a warning when installing the package ([ref](https://github.com/Instawork/hyperview/blob/master/package.json#L117))

Reply:
It's only a dev dependency here, and we won't even be publishing any typescript files, so it won't have any effect when installed in other projects.